lib/vva/errors.rb

```diff
@@ -0,0 +1,27 @@
+module VVA
+  class ClientError < StandardError
+    def initialize(msg = nil)
+      super(msg)
+    end
+
+    def ignorable?
+      false
```

Comment:
Shouldn't we have non dummy implementation here similar to bgs - https://github.com/department-of-veterans-affairs/ruby-bgs/blob/master/lib/bgs/errors.rb#L103?

Reply:
Currently, there doesn't seem to be any errors specific to VVA that we are handling differently. If we come across any, then we will mimic the bgs implementation.